index.js

```diff
@@ -164,9 +167,15 @@ switch (comp.config) {
     break;
 }
 
-// css
-if (comp.css)
-  writeFile(`${comp.fullPath}.css`, getTemplate(comp.template, "css", comp.name))
+// styles
+switch (comp.styles) {
+  case 'scss':
+    writeFile(`_${comp.fullPath}.scss`, getTemplate(comp.template, "scss", comp.name))
```

Comment:
just realized that this underscore causes an error when a component name isn't passed (and grabs the parent directory name)